packages/ui/src/components/editor/panels/Hierarchy/container/index.tsx

```diff
@@ -264,6 +264,40 @@ function HierarchyPanelContents(props: { sceneURL: string; rootEntity: Entity; i
     setContextSelectedItem(undefined)
     setAnchorEvent(undefined)
   }
+  enum OperatingSystems {
```

Comment:
let's move all this new code to module scope in a helper file in `packages/common` so we can reuse it in various places